client/mysqltest.cc

```diff
@@ -10318,17 +10318,20 @@ int insert_pointer_name(reg1 POINTER_ARRAY *pa,char * name)
 
   if (! pa->typelib.count)
   {
+    const size_t type_names_size = ((PC_MALLOC-MALLOC_OVERHEAD)/
+         (sizeof(char *)+sizeof(*pa->flag))*
+         (sizeof(char *)+sizeof(*pa->flag)));
     if (!(pa->typelib.type_names=(const char **)
-	  my_malloc(((PC_MALLOC-MALLOC_OVERHEAD)/
-		     (sizeof(char *)+sizeof(*pa->flag))*
-		     (sizeof(char *)+sizeof(*pa->flag))),MYF(MY_WME))))
+	       my_malloc(type_names_size, MYF(MY_WME))))
       DBUG_RETURN(-1);
+    memset(pa->typelib.type_names, 0, type_names_size);
     if (!(pa->str= (uchar*) my_malloc((uint) (PS_MALLOC-MALLOC_OVERHEAD),
 				     MYF(MY_WME))))
```

Comment:
Likewise

Reply:
Likewise to what?

Reply:
To the comment above which for some reason is gone :) Reposted